test/automation/src/quickaccess.ts

```diff
@@ -171,15 +171,47 @@ export class QuickAccess {
 	}
 
 	async runCommand(commandId: string, keepOpen?: boolean): Promise<void> {
+		let retries = 0;
+		let error;
+
+		while (true) {
+
+			if (++retries > 5) {
+				throw error ?? new Error(`Command: ${commandId} Not found`);
+			}
+
+			// open commands picker
+			await this.openQuickAccessWithRetry(QuickAccessKind.Commands, `>${commandId}`);
+
+			// wait for best choice to be focused
+			await this.quickInput.waitForQuickInputElementFocused();
```

Comment:
I wonder if this line can be removed given the next. See how `openQuickOutline` works similar without.

Reply:
May be not. This is to check for focus so that we can click on best choice next.